Makefile

```diff
@@ -366,6 +387,10 @@ loop: mod core
 	@$(CC) -c -o $(PREFIX)/loop_ping.o test/loop_ping.c $(CFLAGS) -ggdb
 
 units: mod core lib $(UNITS_OBJ) $(MOD_OBJ)
+	@if [ $(UNITS) -eq 0 ]; then \
```

Comment:
that won't work because of the way makefiles are parsed. 

One thing that might work is a target that calls a submake with all the correct flags set, but that's a pretty dirty workaround...